src/frontend/komponenter/Fagsak/Dokumentutsending/DeltBosted/DeltBostedAvtaler.tsx

```diff
@@ -0,0 +1,166 @@
+                                                        return [...acc, dato ?? ''];
+                                                    } else {
+                                                        return [...acc, forrigeAvtaleDato];
+                                                    }
+                                                },
+                                                []
+                                            ),
+                                        }
+                                    );
+                                }}
+                                valgtDato={avtaleDato}
+                            />
+                            {index !== 0 && (
+                                <FjernAvtaleKnapp
+                                    erLesevisning={false}
+                                    id={`fjern_avtale__${barn.ident}`}
+                                    mini={true}
+                                    ikon={<Slett />}
+                                    knappPosisjon={'venstre'}
+                                    onClick={() => {
+                                        deltBostedSkjema.felter.avtalerOmDeltBostedPerBarn.validerOgSettFelt(
+                                            {
+                                                ...deltBostedSkjema.felter
+                                                    .avtalerOmDeltBostedPerBarn.verdi,
+                                                [barn.ident]: avtalerOmDeltBosted.reduce(
+                                                    (
+                                                        acc: string[],
+                                                        forrigeAvtaleDato: string,
+                                                        reduceIndex: number
+                                                    ) => {
+                                                        if (index === reduceIndex) {
+                                                            return acc;
+                                                        } else {
+                                                            return [...acc, forrigeAvtaleDato];
+                                                        }
+                                                    },
+                                                    []
+                                                ),
+                                            }
+                                        );
+                                    }}
+                                    label={'Fjern'}
+                                />
+                            )}
+                        </DatovelgerOgSlettknapp>
+
+                        {feilmelding && <StyledFeilmelding>{feilmelding}</StyledFeilmelding>}
+                    </div>
+                );
+            })}
+
+            {barn.merket && (
+                <LeggTilAvtaleKnapp
```

Comment:
har ikke noe med koden å gjøre, men jeg ville tenkt at denne knappen trigga et slags "legg til-skjema" i likhet med "Legg til barn" som er prikk lik (og fritekst feks?). Men så er det egentlig submit for "avtaleskjema" på det barnet? Eller kan man legge til flere datoer per barn?

Reply:
Denne oppfører seg helt likt som "legg til barn". Det skal være mulig å legge til flere avtaler per barn.